specification/appendix/examples/metadata_examples/adding_new_columns_example.md

```diff
@@ -0,0 +1,79 @@
+# Adding New Columns
+
+## Scenario
+
+ACME has decided add additional columns to their FOCUS data export. The new columns are x_awesome_column1, x_awesome_column2, and x_awesome_column3. The provider creates a new schema object to represent the new schema, this schema object has a unique SchemaId. The subsequent data exports that use the new schema include the new schema's id as a reference to their corresponding schema object.
+
+### Supplied Metadata
+
+#### Location for the new schema object:
+
+`/FOCUS/metadata/schemas/schema-23456-abcde-23456-abcde-23456.json`
+
+#### Content for the new schema object:
+
+```json
+ {
+  "SchemaId": "23456-abcde-23456-abcde-23456",
+  "FocusVersion": "1.0",
```

Comment:
Maybe I'm misunderstanding -- but do they not need to specify `ProviderVersion` here?

Reply:
Provider Version is an optional metadata value and therefore isn't included here.